framework/auth/cas.py

```diff
@@ -285,7 +285,7 @@ def make_response_from_ticket(ticket, service_url):
             # if user is authenticated by CAS
             return authenticate(
                 user,
-                cas_resp.attributes['accessToken'],
+                cas_resp.attributes.get('accessToken', ''),
```

Comment:
Set `accessToken` to empty if service validation does not release it in attributes. Please refer to this ticket: https://openscience.atlassian.net/browse/CAS-27.